Build/Projects/MonoGame.Framework.definition

```diff
@@ -136,6 +136,7 @@
       <!-- Covers compatible GL implementations, such as ANGLE and OpenGL -->
     </Service>
     <Service Name="_XNADesignProvided">
+      <AddDefine>XNADESIGNPROVIDED</AddDefine>
```

Comment:
I saw that...  just wasn't sure what the problem is.

My guess is that previously we were only setting the type converters on Windows builds and with this it now appears on Mac and Linux too?